### DISCUSSION TITLE
How to cite you

### DISCUSSION THREAD

**Discussion by R1ncewind** [sknetwork-team/scikit-network#517] [Q&A] [answered]
Dear scikit-network team,
I am using a few of the methods in your library (to be precise, the first order link prediction ones) for my masters thesis.
Do you have any preferences on how to be cited/credited for this?
I could not find a 'cite this as:' in the documentation.

Thanks in advance

**Comment by R1ncewind** [comment, marked as answer]
And I just found it in the readme of this repository.
Maybe that could be added to the documentation page as well?

**Comment by tbonald** [comment]
Thanks for your comment.
The reference is on the documentation as well:
https://scikit-network.readthedocs.io/en/latest/
Did I miss something?
